test/testdata/slsa/v1/task-output-image.json

```diff
@@ -44,7 +44,7 @@
         "materials": [
 	    {{range .URIDigest}}
 	    {
-		"uri": "{{.URI}}",
+		"uri": "oci://{{.URI}}",
```

Comment:
May be we should capture this in the URI itself. The reason is that materials can also contain repos of the remote tasks and pipelines which could be on git/oci/artifact hub.

Reply:
I think our example tests inline the spec (instead of fetching them remotely) so they are not throwing errors.